pkgs/development/tools/jq/default.nix

```diff
@@ -1,4 +1,5 @@
-{ lib, stdenv, nixosTests, fetchurl, oniguruma }:
+{ lib, stdenv, nixosTests, fetchurl, autoreconfHook
+, oniguruma, onigurumaSupport ? true }:
```

Comment:
```suggestion
, onigurumaSupport ? true, oniguruma }:
```

Reply:
Why it is better?

Reply:
> Why it is better?

It's a matter of taste - it's putting the arguments in their order of significance sort of I think.

Reply:
> It's a matter of taste

yes. I usually add more taste review bits if I already found something which needs to change.

The idea is to put packages behind the flags they are used in.